src/Routes.tsx

```diff
@@ -1,17 +1,17 @@
 import { Route, Routes as RouterRoutes } from 'react-router-dom';
 
-import GuideCategory from './pages/GuideCategory';
 import Main from './pages/Main';
 import TopicDetail from './pages/TopicDetail';
+import GuideSection from './pages/GuideSection';
 
 // TODO: change the naming of the paths - should be in Czech language
 export const Routes = () => {
   return (
     <RouterRoutes>
       <Route index path="/" element={<Main />} />
-      <Route path="/sections/:id" element={<GuideCategory />} />
+      <Route path="/sekce/:id" element={<GuideSection />} />
```

Comment:
Same as above, maybe just Section but I understand that SectionSection is far worse